src/components/DashboardMain.js

```diff
@@ -44,230 +44,80 @@ class Main extends Component {
-                  {this.props.l10n("main.menu")}
-                </a>
-                <button
-                  className="navbar-toggler"
-                  type="button"
-                  onClick={() =>
-                    this.setState({ openTabs: !this.state.openTabs })
-                  }
-                >
-                  <span className="navbar-toggler-icon" />
-                </button>
-              </nav>
-            </div>
-            <Collapse
-              id="eduid-menu-items-xs"
-              isOpen={this.state.openTabs}
-              className="text-center"
-            >
-              <nav className="navbar navbar-light bg-light">
-                <ul className="nav nav-stacked nav-tabs navbar-nav">
-                  {tabsElems("main-nav-tabs tab-link btn-block")}
-                </ul>
-              </nav>
-            </Collapse>
-            {routesElem("")}
-            {profElem}
-          </div>
-        ];
-      }
-    } else {
-      contentElem = routesElem(<NotificationsContainer />);
-    }
-
     return (
       <FetchingContext.Provider value={this.state}>
         <SplashContainer />
         <div className="container-fluid">
           <HeaderContainer />
           <ConnectedRouter history={history}>
             <div id="content-block">
-              <div
-                className="tabbable well profile-combo"
-                id="profile-content-area"
+              <Link
+                className="button"
+                id="submit-button-link"
+                to={`/profile/settings/`}
               >
-                {contentElem}
+                <button
+                  id="settings-button"
+                  type="submit"
+                >
+                  Settings
+                </button>
+              </Link>
+              <div id="dashboard-text">
+                <h1>eduID for email@email.com</h1>
+                <p>
+                  {" "}
+                  Welcome to your eduid account. To be able to use it you
+                  need to provide some more information.
+                </p>
+                <div id="content">
+                  <Route
+                    path="/profile/settings/"
+                    component={SettingsComponent}
+                  />
+                  <div id="welcome">
+                    <div id="verify-identity-prompt">
+                      <h3>
+                        {" "}
+                        You're almost done, the next step is to verify your
+                        identity{" "}
+                      </h3>
+                      <p>
+                        {" "}
+                        Choose a suitable way to verify your identity and
+                        follow the instuctions to start using eduID. You can
+                        change any of your personal information in Settings.
+                      </p>
+                      <div id="verify-identity-button">
+                        <button
```

Comment:
Just in case you did not notice, there's an EduIDButton (in components/EduIDButton.js), which is a normal bootstrap button (can be given any of bootstrap's button classes), with the peculiarity that when there is some async process going on (such as fetching some data from the server) all EduIDButtons with be disabled and will have a spinner.
I'm not saying it should be used here, use it at your own discretion, I'm just saying in case you had not noticed.

Reply:
Thanks for letting me know. I will check it out! I do have a specific solution for this button (coming later today!), but do let me know in that review if you think the EduIDButton component would be better suited...

Reply:
perfect,looking forward to it!